traceloader.py

```diff
@@ -82,33 +82,47 @@ class TraceLoader:
     def __init__(self,
                  trace_list: list,
                  trace_folder: str,
-                 link_trace_save=1,
-                 raw_save=1,
-                 trace_info_save=1):
-        '''
-        initialize the traceloader
-
+                 link_trace_save=True,
+                 raw_save=True,
+                 trace_info_save=True):
+        '''[summary]
```

Comment:
This `[summary]` should be remove. It is the first line you conclude the function.